app/src/main/java/se/nullable/flickboard/model/layouts/FRPuncMessagEase.kt

```diff
@@ -19,6 +19,7 @@ val FR_PUNC_MESSAGEASE_MAIN_LAYER = Layer(
                     Direction.TOP_RIGHT to Action.Text("à"),
                     Direction.BOTTOM_RIGHT to Action.Text("v"),
                     Direction.LEFT to Action.Text("«"),
+                    Direction.TOP_LEFT to Action.Text("æ"),
```

Comment:
I would prefer leaving a-topleft free for eventually implementing #72 :/

Maybe there's something else it could be squeezed in?

Reply:
On MessagEase, it's precisely this © key that transforms O and Œ and A into Æ!

Issue #72 could therefore solve my problem, is it possible to pay attention that the Æ and Œ are well implemented please?

Reply:
How long do you think it will take to implement ©?
If that's a long time, what do you think if we validate my changes temporarily, and the day the © is implemented, we remove these two characters?

Reply:
Æ is actually already implemented via Zalgo mode: a, enter-down, e. Admittedly not super convenient though.

Oe isn't currently registered there, but should be easy enough to add.

Reply:
@nightkr Thanks for the discovery!
There should be a pictogram on the enter-down, I didn't know it existed.

Reply:
Happy to hear if you have any suggestion!

Reply:
@nightkr This pictogram ?
https://pictogrammers.com/library/mdi/icon/table-merge-cells/

Reply:
https://github.com/nightkr/flickboard/commit/c7b38bfe2bdf691a88f214f80b5f32b94b0b6bfd